### PR TITLE
CSP-380-Tribal-ProviderSearch

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/SFA.DAS.Roatp.ProviderModeration.Web.UnitTests/MockedObjects/MockedUser.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using System.Security.Claims;
+
+namespace SFA.DAS.Roatp.ProviderModeration.Web.UnitTests.MockedObjects
+{
+    public class MockedUser
+    {
+        private const string GivenName = "Test";
+        private const string Surname = "User";
+        private const string Email = "Test.User@example.com";
+        private const string RoleClaimType = "http://service/service";
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.ProviderModeration.Web.UnitTests/MockedObjects/MockedUser.cs:11]: you might need to change this to https to get through the security hotspot, but not 100% sure about it.  We can always wave it through I guess.

File: src/SFA.DAS.Roatp.ProviderModeration.Application/Queries/GetProvider/GetProviderQuery.cs [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
-    public int Ukprn { get; init; }
+    public class GetProviderQuery : IRequest<GetProviderQueryResult>
+    {
+        public int Ukprn { get; }
+
+        public GetProviderQuery(int ukprn)
+        {
+            Ukprn = ukprn;
+        }
+    }
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.ProviderModeration.Application/Queries/GetProvider/GetProviderQuery.cs:13]: add test to check setup

**Review by MarkFCain — COMMENTED** [review]
This all looks fine and I will approve once the sonarcloud code smells around the css are sorted out

File: src/SFA.DAS.Roatp.ProviderModeration.Web.UnitTests/MockedObjects/MockedUser.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.ProviderModeration.Web.UnitTests/MockedObjects/MockedUser.cs]: Change it back to http please

**Review by MarkFCain — APPROVED** [review]

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**